src/client.js

```diff
@@ -3,11 +3,22 @@ import React from 'react';
 import { hydrate } from 'react-dom';
 import { ClientApp } from './app/containers/App';
 import routes from './app/routes';
+import {
+  reloadAfterInactivity,
+  resetInactivityTimer,
+} from './invalidationStrategy';
+
+reloadAfterInactivity({
+  hours: 1,
```

Comment:
To test this, you can change this line to `minutes: 1` locally and observe the refresh happening.

But I think there's also value in leaving it as-is, in case our implementation of `hours: 1` is incorrect or to catch if any issues arise 20/30/40 minutes in which stop the refresh from happening! Will leave it up to you 😄